### PR TITLE
rats-tls: add support to connect verdictd based on EAA echo command

### PR DESCRIPTION
**Pull request by haosanzi**
Signed-off-by: Shirong Hao <shirong@linux.alibaba.com>

### CODE REVIEW

File: rats-tls/samples/sgx-stub-enclave/sgx_stub_ecall.c [hunk 1]
```diff
@@ -200,7 +200,12 @@ int ecall_rtls_client_startup(sgx_enclave_id_t enclave_id, rats_tls_log_level_t
 		goto err;
 	}
 
-	const char *msg = "Hello and welcome to RATS-TLS!\n";
+	const char *msg;
+	if (verdictd)
+		msg = "{ \"command\": \"echo\", \"data\": \"Hello and welcome to RATS-TLS!\" }";
```
Comment (zhiminghufighting) [inline rats-tls/samples/sgx-stub-enclave/sgx_stub_ecall.c:205]: please add "n" behind the last "\".

File: rats-tls/samples/rats-tls-client/client.c [hunk 2]
```diff
@@ -143,7 +143,12 @@ int rats_tls_client_startup(rats_tls_log_level_t log_level, char *attester_type,
 		goto err;
 	}
 
-	const char *msg = "Hello and welcome to RATS-TLS!\n";
+	const char *msg;
+	if (verdictd)
+		msg = "{ \"command\": \"echo\", \"data\": \"Hello and welcome to RATS-TLS!\" }";
```
Comment (zhiminghufighting) [inline rats-tls/samples/rats-tls-client/client.c:148]: Can wen remove \"data\" and just keep the msg as "command: echo: XXXX"? That will be more brief.

Comment (zhiminghufighting) [inline rats-tls/samples/rats-tls-client/client.c:148]: please add "n" behind the last "\" to keep align with the msg format with the original msg.

Comment (haosanzi) [reply rats-tls/samples/rats-tls-client/client.c:148]: Thank you for your comments. 
The communication between `rats-tls-client` and `verdictd` is based on EAA protocol.
The echo command in EAA protocol is defined as followed:
```JSON
{
    "command": "echo"
    "data": "xxxxxxx"
}
```

**Review by zhiminghufighting — APPROVED** [review]

**Pull request merged**